src/components/EntryList.js

```diff
@@ -14,7 +14,7 @@ const EntryList = ({ entries, onDelete, getEntry, onUpdate }) => {
     <Container>
       <Grid.Row>
         <Grid.Column>
-          <Header as="h1">Entries</Header>
+          <Header as="h2">Entries</Header>
```

Comment:
How come this was changed from `h1` to `h2`?

Reply:
It was purely a design choice. I thought an `h1` looked too big in relation to the other elements on the page.